### PR TITLE
add 1.9.5 release notes

### PR DESCRIPTION
**Pull request by sascala**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

File: src/releases/1.9.5/notes.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+<a name="1-9-4"></a>
+# Issues Fixed Since 1.9.3
+
+- MESOS-7926 - Abnormal termination of default executor can cause MesosContainerizer::destroy to fail.
+- MESOS-7934 - OOM due to LibeventSSLSocket send incorrectly returning 0 after shutdown.
+
+<a name="1-9-5"></a>
+# Issues Fixed in 1.9.5
+
+- CORE-1292: Remove the systemd prereq requirement of `leader.mesos` for Mesos agent.
```
Comment (judithpatudith) [inline src/releases/1.9.5/notes.md]: The delimiter between the ticket number and the ticket name is not consistent. Sometimes ":" sometimes "_-_" and sometimes "-_".

File: src/releases/1.9.5/notes.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+- DCOS_OSS-1301 - Exhibitor doesn't log multiline messages to journald.
+- DCOS_OSS-1433 - Bug in /etc/systemd/system/dcos-docker-gc.service script (1.9).
+- DCOS_OSS-1451 - Stale datapoints can occur in container metrics.
+- DCOS_OSS-1466 - Marathon returns persistent 503's.
+- DCOS_OSS-1470 - Exhibitor: use PatternLayoutEscaped logger layout for structured journal logger.
+- DCOS_OSS-1486 - Metrics agent crashes when the mesos containers endpoint is missing fields.
+- DCOS_OSS-1561 - Revert DCOS_OSS-1472 from 1.9.3.
+
+<a name="1-9-4"></a>
+# Issues Fixed Since 1.9.3
```
[the remark this answers is not in the document]
Comment (sascala) [reply src/releases/1.9.5/notes.md]: why yes

File: src/releases/1.9.5/notes.md (hunk 1 above)
Comment (sascala) [reply src/releases/1.9.5/notes.md]: thanks for noticing that

File: src/releases/1.9.5/notes.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+- DCOS-14005 - Marathon-LB does not support pods.
+- DCOS-14021 - [Task logging to journald](/1.9/monitoring/logging/) disabled by default, so task logs will continue to be written to their sandboxes, and logrotated out. The `dcos task log` command will work as it did before.
+- Marathon-7133 - Marathon application history is lost after Marathon restart.
+- CORE-1191 -  The Mesos master's event queue can get backlogged with the default settings, causing performance problems. These can be mitigated by setting the following configuration parameter in your `config.yaml` file at install time. See the [Configuration Reference](/1.9/installing/custom/configuration/configuration-parameters/) for more information. **Note:** Lowering this parameter also reduces the number of tasks per framework that the `dcos task` subcommands can access for debugging. If you run a framework with many short tasks, such as Spark, you may not want to reduce this value.
+
+  ```yaml
+  mesos_max_completed_tasks_per_framework: 20
+  ```
+
+# <a name="fixed-issues"></a>Issues Fixed in 1.9
```
Comment (judithpatudith) [inline src/releases/1.9.5/notes.md]: Where did 1.9.1 go?

**Pull request merged**